new/PRS13150WalattaM.xml

```diff
@@ -0,0 +1,62 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13150WalattaM" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Walatta Madḫǝn Ḍaḥaytu</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2020-09-15">Created entity</change>
+        </revisionDesc>
+    </teiHeader>    
+    <text>
+        <body>
+            <listPerson>
+                <person sex="2">
+                    <persName xml:lang="gez" xml:id="n1">ወለተ፡ መድኅን፡ ፀሐይቱ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Walatta Madḫǝn Ḍaḥaytu</persName>
+                    <persName><roleName type="title">wayzaro</roleName></persName>
+                    <floruit notBefore="1800" notAfter="2006"/>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS13150WalattaM" passive="BDLaethg37"/>
+                    <relation name="dcterms:relation" active="PRS13150WalattaM" passive="PRS13153Taytu"><desc><persName ref="PRS13150WalattaM"/>
```

Comment:
I don't understand: WalattaM und Taytu are similar names?

Reply:
Her full name is apparently Walatta Madḫǝn Ḍaḥaytu; and Ḍaḥaytu and Ṭāytu are etymologically related - but I don't know naming practices well enough to be able to tell whether they might refer to the same person.

Reply:
But Taytu stands alone, yes?

Reply:
Yes, they are mentioned in two different note, both crudely written, but probably not by the same hand